run_unittests.py

```diff
@@ -4193,4 +4201,6 @@ def should_run_cross_mingw_tests():
     if is_osx():
         cases += ['DarwinTests']
 
+    cases = ['LinuxlikeTests.test_generate_gir_with_address_sanitizer']
```

Comment:
Typo?

Reply:
You can run it with `./run_unittests.py LinuxlikeTests.test_generate_gir_with_address_sanitizer`. It's documented in the `--help` output, do you have suggestions on where else to document it?

Reply:
Ah OK, yeah it should probably go in `docs/markdown/Contributing.md`